### PR TITLE
Add image map to Edinburgh sketch, with links to buildings.

### PR DESCRIPTION
**Pull request by lornaarmstrong**
Add image map to the building drawings so that, when clicked on, a new tab opens to a relevant website for the building / place, offering further information.

Next, add the explanation of informing user to click on images.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -44,9 +44,24 @@ <h3>Fun Facts</h3>
...
         <p>Here is a sketch of mine, showing some of my favourite places in Edinburgh.</p>
       </section>
+
       <div class="img-nonprofile">
         <img src= "/images/edinburgh-sketch.png" alt="Edinburgh Buildings Sketch" 
-        width=480px height=333px margin-right=15px>
+        width=480px height=333px margin-right=15px usemap="#buildingmap">
+        <map name="buildingmap">
+            <area shape="rec" coords="23,43,133,119" 
+            href="http://www.pilgrimbar.co.uk/" alt="Pilgrim Bar" target="_new">
```
Comment (KaziZubaer) [inline portfolio/src/main/webapp/index.html:53]: Using `target="_blank"` seems to be the preferable choice ([ref](https://stackoverflow.com/q/4964130))

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -16,7 +16,7 @@ body {
 h1 {
   padding: 20px 20px 20px 0px;
   text-align: center;
-  background-color:lightgray;
+  background-color:tan;
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/style.css:19]: nit: leave a blank space after colon please.

Comment (MostafaGabriel) [inline portfolio/src/main/webapp/style.css:19]: nit: leave a blank space after colon please.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/index.html:53]: Thanks Kazi!

**Pull request merged**